spec/amp-html-format.md

```diff
@@ -170,7 +170,7 @@ In concrete terms this means that:
 AMP HTML documents MUST
 
 - <a name="dctp"></a>start with the doctype `<!doctype html>`. [🔗](#dctp)
-- <a name="ampd"></a>contain a top-level `<html ⚡>` tag (`<html amp>` is accepted as well). [🔗](#ampd)
+- <a name="ampd"></a>contain a top-level `<html ⚡ lang="en">` tag (`<html amp>` is accepted as well). [🔗](#ampd)
```

Comment:
Please remove from this line, since `lang="en"` is not a "MUST"

Reply:
done